scripts/run.py

```diff
@@ -37,6 +39,10 @@ def main():
         storage.store(args.output_path, ext_ids)
         return
 
+    if 'pipe2' == args.step:
```

Comment:
I'd prefer to see a less generic name for the step, maybe something like 'scrape' which clearly describes the main action

Reply:
Dune:
![image](https://user-images.githubusercontent.com/5351225/80691137-206bfc80-8ad0-11ea-9dfc-24a3aa00302e.png)